drivers/wireless/gs2200m.c

```diff
@@ -1730,12 +1730,12 @@ enum pkt_type_e gs2200m_get_wstatus(FAR struct gs2200m_dev_s *dev)
 
 /****************************************************************************
  * Name: gs2200m_create_tcpc
- * NOTE: See 7.5.1.1 Create TCP Clients
+ * NOTE: See 7.5.1.1 Create TCP Clients and 7.5.2.2 Create UDP Client
```

Comment:
do you mean 7.5.1.2 ?

Reply:
> do you mean 7.5.1.2 ?

You are right, I'll fix the comment later.

Reply:
Just fixed and pushed -f.